docs/user-guide/secrets/index.md

```diff
@@ -214,6 +214,52 @@ You can package many files into one secret, or use many secrets, whichever is co
 
 See another example of creating a secret and a pod that consumes that secret in a volume [here](/docs/user-guide/secrets/).
 
+##### Projection of secret keys to specific paths
```

Comment:
There are already level-5 headings on the site. At the same time, this part must go under "Using Secrets as Files from a Pod" which is level-4 heading already.